engine/src/test/java/com/redhat/cloud/notifications/processors/camel/google/chat/GoogleChatTestProfile.java

```diff
@@ -0,0 +1,6 @@
+package com.redhat.cloud.notifications.processors.camel.google.chat;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+public class GoogleChatTestProfile implements QuarkusTestProfile {
```

Comment:
Why do we need a specific test profile for each integration ?
As far I remember, we had an issue on unit tests because we were using identical mock urls for each tests.
If it's the only reason why a test profile is required, I think we should use a dedicated target url like we had before to save running test time ?

Reply:
I added back the test profiles because the tests no longer worked after mocking the source Kafka endpoint. This is not related to the mock URL change. The same URL can be used for everything as long as `getClient().reset()` is called between each test.

Reply:
The test profiles will be removed very soon when each connector will be moved to its own Maven module.

Reply:
Ok then if it's temporary

Reply:
Ok then if it's temporary